lib/db/schema/users.ts

```diff
@@ -15,6 +15,7 @@ export const users = pgTable('users', {
   passwordHash: text('password_hash'),  // Make password optional for OAuth users
   email: varchar('email', { length: 255 }).notNull().unique(),
   profileImageId: integer('profile_image_id').references(() => photos.id),
+  bio: text('bio'),
```

Comment:
Was bio mentioned in the documentation we prepared last semester? If not, we should add it, I guess.

Reply:
it was mentioned, page 51